core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -746,6 +746,21 @@ paths:
       summary: list active submissions
       description: List all active submissions in all workspaces
       operationId: listAllActiveSubmissions
+      parameters:
+        - name: startDate
+          in: query
+          description: Optional start date to filter results (inclusive). Default value is 30 days ago.
+          schema:
+            type: string
+            format: date
+            example: 2025-10-01
+        - name: endDate
+          in: query
+          description: Optional end date to filter results (inclusive). Default value is the current date.
+          schema:
+            type: string
+            format: date
+            example: 2025-10-31
```

Comment:
If the user omits these, we need to default to returning all submissions i.e. no date filtering. Otherwise, we're changing behavior of this API for scripters, etc. and we decided we don't want to do that.

Reply:
Oh you're right! I mixed this up with the UI defaulting to the last 30 days

Reply:
Ah I see you added these query params to `/api/workspaces/{workspaceNamespace}/{workspaceName}/submissions` below ... they should get removed from `/api/admin/submissions` here, since we haven't updated that endpoint.

Reply:
I think it would be nice to have that for consistency, so I'll make a follow-on ticket for it